.github/workflows/pr-comment.yml

```diff
@@ -0,0 +1,12 @@
+name: PR Comment
+
+on: [pull_request_target]
+
+jobs:
+  pr-comment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: angelikatyborska/pr-commenter-action@97e4da967b377bf97ce50bfa102a39c8decf4ace
```

Comment:
OOC why does your action run on node 12 vs node 14?

Reply:
I was following https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action when creating the action and copying some code from https://github.com/actions/labeler/blob/main/action.yml. They both mention node12, so assumed that's what I should do... I'm not really sure why or what would be the consequences of bumping to node14 instead. Do you know? Does it matter?

Reply:
It might not matter for this, and depending how `ncc` handles the code it might not matter at all.  I think there are a vew advantages to v14 over v12.  v14 more security updates/bug fixes.  Also features V8 8.1 javascript runtime engine, so gives native access to things like nullish coalescing `??` and optional chaining `?.`, which makes working with objects that much nicer.

Reply:
If this won't affect the (future) users of the action, I'll switch to the newer node.

Reply:
🤷🏻 I'm not sure you have to change it at all if you're not using those features or if they are transpiled with `ncc`.  It was just a question to learn about how your action works. 😄 (It looks awesome, btw 🎉 )